src/main/scala/nl/knaw/dans/easy/license/Command.scala

```diff
@@ -15,16 +15,92 @@
  */
 package nl.knaw.dans.easy.license
 
-import org.slf4j.LoggerFactory
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream, FileOutputStream, OutputStream}
+
 import nl.knaw.dans.easy.license.{CommandLineOptions => cmd}
+import org.slf4j.LoggerFactory
+import rx.lang.scala.{Observable, ObservableExtensions}
+import rx.lang.scala.schedulers.IOScheduler
+import rx.schedulers.Schedulers
+
+import scala.collection.JavaConverters._
+import scala.language.postfixOps
+
+class Command(datasetLoader: DatasetLoader,
+              placeholderMapper: PlaceholderMapper,
+              templateResolver: TemplateResolver,
+              pdfGenerator: PdfGenerator)(implicit parameters: Parameters) {
```

Comment:
Agree with you, even though other projects like `SplitMultiDeposit` call it `Main.scala` which require to be renamed then as well. However, I remember discussing this with @janvanmansum [here](https://github.com/DANS-KNAW/easy-module-archetype/pull/2#discussion_r55196318) where I suggested `Main` instead of `Command`. He explained the rational behind it, but maybe I think your suggestion is even better, especially when you're using the projects as dependencies on other projects (#name-collisions).

@janvanmansum can you comment on this?